docs(readme): command explination

jpbnetley opened the pull request:
<!--
Thank you for proposing a pull request! Please note that SOME TESTS WILL
LIKELY FAIL due to how GitHub exposes secrets in Pull Requests from forks.
Someone from the team will review your Pull Request and respond.

Please describe your change and any implementation details below.
-->
- added explanation to the configuration section, explaining what some of the properties do.

Not sure if there is a better section to explain the input commands.
When I read the documentation, I didn't quite understand what those 2 commands do.

I ended up trying to follow this [example](https://github.com/conventional-changelog/conventional-changelog/actions/runs/1635244766/workflow) to see what they do.

Please let me know if there is a better way to word the changes I made.

andrey-yantsen commented on README.md:
```diff
@@ -66,6 +66,15 @@ Automate releases with Conventional Commit Messages.
+#### Configeration definitions
+##### `Command` definition
+> Some additional info regardng the `command` property.
+- `github-release` command creates github release notes (as mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)), that incluids the tag versons and the commits for those tags as release notes. 
```
There're a couple of typos: **incluids** and **versons**.

```suggestion
- `github-release` command creates github release notes (as mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)), that includes the tag versions and the commits for those tags as release notes. 
```

jpbnetley replied:
aah. Nice spot.
My apologies, thank you for the feedback.
Updates applied.

jpbnetley commented:
@andrey-yantsen I tried re-request your review, but it just refreshes.
Could you please review when you have a chance?
:)

andrey-yantsen commented:
@jpbnetley never mind me, I'm just passing by and have no write access to the repo :)

bcoe commented on README.md:
```diff
@@ -66,6 +66,15 @@ Automate releases with Conventional Commit Messages.
 | `repo-url` | configure github repository URL. Default `process.env.GITHUB_REPOSITORY` |
 | `github-graphql-url` | configure github GraphQL URL. Default `https://api.github.com` |
 
+#### Configeration definitions
+##### `Command` definition
```
I suggest making this heading:

```
### The `command` property
```

bcoe reviewed (changes requested):
Left a couple suggestions.

jpbnetley commented:
Thank you @bcoe. I have applied the feedback. When you get a chance, please review and let me know if how it goes :)

bcoe commented on README.md:
```diff
@@ -66,17 +66,10 @@ Automate releases with Conventional Commit Messages.
-| `pr` | The PR number of an opened release (undefined if no release created) |
+### The `command` property
+> Some additional info regarding the `command` property.
+- `github-release` command creates github release notes (as mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)), that includes the tag versions and the commits for those tags as release notes. 
```
```suggestion
- `github-release`: creates GitHub releases (as mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)) based on the most recently merged release PR and the release strategy being used.```

bcoe commented on README.md:
```diff
@@ -66,17 +66,10 @@ Automate releases with Conventional Commit Messages.
+### The `command` property
+> Some additional info regarding the `command` property.
+- `github-release` command creates github release notes (as mentioned [here](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)), that includes the tag versions and the commits for those tags as release notes. 
+- `release-pr` uses conventional commits to create a git tag, and opens a [pull request](#how-release-please-works). Once the pull request is merged, this property will push the tags to the repo.
```
```suggestion
- `release-pr`: uses Conventional Commits to propose a candidate release [pull request](#how-release-please-works). This pull request, once merged, is used by `github-release`/`manifest````

bcoe reviewed (changes requested):
If we're going to document `github-releasea` and `release-pr`, we should also document `manifest` and `manifest-pr`.

https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#manifest

jpbnetley commented:
> If we're going to document `github-releasea` and `release-pr`, we should also document `manifest` and `manifest-pr`.
> 
> https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#manifest

I added the above mentioned docs.
I am not 100% sure if the explanation is explains what the manifest files do.
(I also added a reference to the link you mentioned. 

If there is maybe a better way to explain it, please let me know :)
